### PR TITLE
Create templates

### LINKED ISSUE #7: Create templates for the tooltip

### PR DESCRIPTION
**Pull request by rodfersou**
closes #7

### CODE REVIEW

File: src/collective/richpreview/browser/richpreview.pt [hunk 1]
```diff
@@ -5,4 +5,23 @@
      i18n:domain="collective.richpreview"
      i18n:attributes="data-loading-message loading_message;
                       data-error-message error_message">
+  <div class="richpreview-template hiddenStructure"
+       i18n:translate="loading-message">
```
Comment (hvelarde) [inline src/collective/richpreview/browser/richpreview.pt]: `loading_message`

File: src/collective/richpreview/browser/richpreview.pt [hunk 2]
```diff
@@ -5,4 +5,23 @@
...
+    Loading page information…
+  </div>
+
+  <div class="richpreview hiddenStructure">
+    <div class="richpreview-image">
+      <img class="richpreview-image"
+           src="#"
+           alt="empty image"
```
Comment (hvelarde) [inline src/collective/richpreview/browser/richpreview.pt]: ` `

File: src/collective/richpreview/browser/richpreview.pt [hunk 3]
```diff
@@ -5,4 +5,23 @@
...
+           />
+    </div>
+    <div class="richpreview-text">
+      Page title
+    </div>
+    <div class="richpreview-link">
```
Comment (hvelarde) [inline src/collective/richpreview/browser/richpreview.pt]: I think this doesn't make any sense; just remove it.

File: webpack/app/js/richpreview.js [hunk 4]
```diff
@@ -54,21 +53,21 @@ export default class RichPreview {
...
-          </div>
-        </div>
-      `;
+
+      let $template = $('#viewlet-richpreview .richpreview').clone();
+      $template.removeClass('hiddenStructure');
+
+      let $image = $('.richpreview-image > img', $template);
+      $image.attr('src', data.image);
+      $image.attr('alt', data.description);
```
Comment (hvelarde) [inline webpack/app/js/richpreview.js]: don't use the description here.

File: webpack/app/js/richpreview.js [hunk 5]
```diff
@@ -54,21 +53,21 @@ export default class RichPreview {
...
-          <div class="richpreview-text">
-            ${data.title}
-          </div>
-          <div class="richpreview-link">
-            <a href="${tip.reference.href}">Link</a>
-          </div>
-        </div>
-      `;
+
+      let $template = $('#viewlet-richpreview .richpreview').clone();
```
Comment (hvelarde) [inline webpack/app/js/richpreview.js]: why do you need to clone it? why you don't work on the original DOM element?

File: src/collective/richpreview/browser/richpreview.pt [hunk 6]
```diff
@@ -5,4 +5,23 @@
      i18n:domain="collective.richpreview"
      i18n:attributes="data-loading-message loading_message;
```
Comment (hvelarde) [inline src/collective/richpreview/browser/richpreview.pt]: do you still need this?

File: src/collective/richpreview/browser/richpreview.pt (hunk 2 above)
Comment (rodfersou) [reply src/collective/richpreview/browser/richpreview.pt]: this way the validators will complain, right?

File: src/collective/richpreview/browser/richpreview.pt (hunk 3 above)
Comment (rodfersou) [reply src/collective/richpreview/browser/richpreview.pt]: why?

File: webpack/app/js/richpreview.js (hunk 4 above)
Comment (rodfersou) [reply webpack/app/js/richpreview.js]: so should I remove the description from the ajax view?

File: src/collective/richpreview/browser/richpreview.pt (hunk 2 above)
Comment (hvelarde) [reply src/collective/richpreview/browser/richpreview.pt]: which validators will complain?

File: webpack/app/js/richpreview.js (hunk 4 above)
Comment (hvelarde) [reply webpack/app/js/richpreview.js]: no, you should use it as the `tiltle` attribute of the anchor on the title.

File: src/collective/richpreview/browser/richpreview.pt (hunk 3 above)
Comment (rodfersou) [reply src/collective/richpreview/browser/richpreview.pt]: I prefer to keep this into template and javascript; If I remove the ``href`` attribute update from the javascript I can't see a way to do it later in a package override.

Knowing this I'll just hide with css this link, ok?

**Pull request merged**